oneview_redfish_toolkit/api/zone.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, profile_template, available_targets_obj, drives=[]):
         self.redfish["Links"]["ResourceBlocks"] = list()
 
         self.fill_resource_blocks(available_targets, drives)
+        self._fill_network_resource_block(profile_template)
```

Comment:
It seems this call should be executed inside fill_resource_blocks() that already handles hardware and drives.